osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/application/services/FirmwareService.java

```diff
@@ -4,11 +4,11 @@
 
 package org.opensmartgridplatform.adapter.protocol.dlms.application.services;
 
-import java.math.BigInteger;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
 import java.util.List;
 import lombok.extern.slf4j.Slf4j;
+import org.bouncycastle.util.encoders.Hex;
```

Comment:
Is dit bewust om verschillende 'Hex-en' te gebruiken? org.apache.commons.codec.binary.Hex 
org.bouncycastle.util.encoders.Hex

Reply:
Yes, it is on purpose. Bouncycastle is more commonly used in the dlms protocol adapter. In essence both libraries do the same.